objects/authentication_factor.json

```diff
@@ -0,0 +1,82 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
+  "name": "authentication factor",
+  "extends": "object",
+  "attributes": {
+    "factor_device": {
+      "description": "Device used to complete the authentication request.",
+      "requirement": "recommended"
+    },
+    "factor_type": {
```

Comment:
1. `enum` fields are often paired with an `id` field in OCSF, where the enum value is actually the `_id` field and the non-`_id` field captures the caption of the enum (most of the time).  Should we follow that pattern here?
2. Many enums in OCSF that have schemas subject to change or interpretation often have a `0` enum for "unknown" and a `99` enum for `Other`.  I think it's probably good to follow that pattern here, as there might be factors others are using now or in the future we haven't thought about

Reply:
Updated for both comments!